pkg/operator/ceph/csi/csi.go

```diff
@@ -317,5 +325,16 @@ func (r *ReconcileCSI) setParams(ver *version.Info) error {
 
 	CSIParam.DriverNamePrefix = k8sutil.GetValue(r.opConfig.Parameters, "CSI_DRIVER_NAME_PREFIX", r.opConfig.OperatorNamespace)
 
+	_, err = r.context.ApiExtensionsClient.ApiextensionsV1().CustomResourceDefinitions().Get(context.TODO(), "volumegroupsnapshotclasses.groupsnapshot.storage.k8s.io", metav1.GetOptions{})
+	if err != nil && !kerrors.IsNotFound(err) {
+		return errors.Wrapf(err, "failed to get volumegroupsnapshotclasses.groupsnapshot.storage.k8s.io CRD")
+	}
+	CSIParam.VolumeGroupSnapshotSupported = err == nil
```

Comment:
Nit suggestion to improve readability. I was confused by this line for a bit.

```suggestion
	CSIParam.VolumeGroupSnapshotSupported = (err == nil)
```